src/components/DatasetCounter.tsx

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+"use client";
+
+import React, { useState, useEffect } from "react";
+import { datasetCount } from "@/services/ckan/index.server";
+
+const DatasetCounter = () => {
+  const [count, setCount] = useState<number | null>(null);
+
+  useEffect(() => {
+    const fetchDatasetCount = async () => {
+      try {
+        const count = await datasetCount();
+        console.log(count);
+        setCount(count);
+      } catch (error) {
+        console.error("Error fetching dataset count:", error);
```

Comment:
Please check with @inderps . We have a component presents errors and exceptions to the user.

Reply:
In this specific case, I think instead of crashing the whole app or console logging anything, i would suggest to show a message saying: "Failed to fetch"